### PR TITLE
Fix error during running on nvptx with cuda9

### PR DESCRIPTION
**Pull request by nishi-t**
This PR fix a error caused by changing `libdevice` file name in CUDA9. I locally confirmed to pass unit tests on CUDA9. 

reference:
https://devtalk.nvidia.com/default/topic/1025985/fedora-25-cuda-9-missing-libdevice-files/?offset=1

### CODE REVIEW

File: python/tvm/contrib/nvcc.py [hunk 1]
```diff
@@ -104,6 +104,23 @@ def find_cuda_path():
     raise RuntimeError("Cannot find cuda path")
 
 
+def get_cuda_version(cuda_path):
+    """Utility function to get cuda version
+
+    Returns
+    -----
```
Comment (tqchen) [inline python/tvm/contrib/nvcc.py]:
```
Returns
-------
```

Comment (tqchen) [reply python/tvm/contrib/nvcc.py]: make sure the lines cover entire returns

Comment (tqchen) [reply python/tvm/contrib/nvcc.py]: please document all the arguments

**Comment by nishi-t** [comment]
@tqchen thank you for your review. I addressed it and locally confirmed to pass tests in cuda9 again because of the mistake included in earlier my commit.

**Comment by tqchen** [comment]
Thanks for the contribution! this is now merged

**Pull request merged**